pkgs/applications/misc/deco/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, scsh, feh, xlibs }:
```

Comment:
You can also pass in `xdpyinfo` if you change the `callPackage` invocation in `all-packages.nix` to `callPackage ... { inherit (xlibs) xdpyinfo; }`.